themes/hugo_theme_robust/static/css/styles.css

```diff
@@ -40,17 +40,24 @@ caption {
   line-height:1.5em;
 }
 
-.hljs-string{
-  color:#183691;
+.hljs-number{
+  color:#aa5146  !important;
```

Comment:
It is usually a sign that something is wrong if you need to add `!important` in css.  Do you know what might be at issue here?

Reply:
there's some other css built into Hugo somewhere that is tacked on at the end (or so it seems). Something called `solarized_dark.min.css`